### PR TITLE
Update ChargebackResponse.cs

### PR DESCRIPTION
**Pull request by Sippelberg**
Added the ChargebackResponseReason parameter to the ChargebackResponse Class

### CODE REVIEW

**Comment by Viincenttt** [comment]
Not sure how this code would work. I can see in the Mollie documentation that there is a reason property, however, it's name is "reason" and it's not of the type ChargebackResponse

**Comment by Sippelberg** [comment]
Hey Viincenttt,
Thanks you very much for the reply.

Since the costs may vary if a chargeback is for example of the type MD06 (€7.5) or MS03(€0.05), and I want to be able to charge these costs to my customer I need to know the response reason.

The Mollie ChargebackResponse object in JSON contains:
      "reason": {
          "code": "MD06",
          "description": "Return of funds requested by end customer"
      },

OR
    
      "reason": {
          "code": "MS03",
          "description": "Reason not specified"
      },

In the code I would use it like this:

ListResponse <ChargebackResponse> myChargebacks = await MollieData.TryGetChargebacks(aTransactie.MollieId);

foreach ( var myChB in myChargebacks.Items ) {
    if (myChB.Reason != null ) {
        mybedrag = mybedrag + SDD_codes.RetrieveForCode(myChB.Reason.code).kosten;
    }
}

In the example I get the costs for the chargeback from a local table "SDD_codes" since Mollie hasn't provided us with an endpoint to get the fees. I opened a support question and they are looking into it but I'm not quite sure if or when they will provide this endpoint.

File: Mollie.Api/Models/Chargeback/ChargebackResponse.cs [hunk 1]
```diff
@@ -32,7 +32,12 @@ public class ChargebackResponse : IResponseObject {
         /// <summary>
         /// The id of the payment this chargeback belongs to.
         /// </summary>
-		public string PaymentId { get; set; }
+	public string PaymentId { get; set; }
+	    
+	/// <summary>
+        /// The reason given for a Chargeback, this can help determine the cost for the chargeback
+        /// </summary>
+        public ChargebackResponse ChargebackResponseReason { get; set; }
```
Comment (Viincenttt) [inline Mollie.Api/Models/Chargeback/ChargebackResponse.cs:40]: This property should be of type `ChargebackResponseReason` not `ChargebackResponse` right?

**Review by Viincenttt — CHANGES_REQUESTED** [review]

File: Mollie.Api/Models/Chargeback/ChargebackResponse.cs [hunk 2]
```diff
@@ -32,7 +32,12 @@ public class ChargebackResponse : IResponseObject {
         /// <summary>
         /// The id of the payment this chargeback belongs to.
         /// </summary>
-		public string PaymentId { get; set; }
+	public string PaymentId { get; set; }
```
Comment (Viincenttt) [inline Mollie.Api/Models/Chargeback/ChargebackResponse.cs:35]: Can you fix the spacing issues in this file? :)

**Review by Sippelberg — COMMENTED** [review]
issues (spacing, ChargebackResponseReason) were resolved

**Review by Viincenttt — APPROVED** [review]
Thanks for making the adjustments!

**Pull request merged**